test/snapshot/list.test.js

```diff
@@ -31,4 +36,26 @@ describe("Test list expressions", function() {
     });
     expect(ast).toMatchSnapshot();
   });
-});
\ No newline at end of file
+
+  it("fix #174", function() {
+    expect(
+      parser.parseEval("list(, $foo,,, $bar,) = $array;")
+    ).toMatchSnapshot();
+  });
+
+  it("fix #174 (2)", function() {
+    expect(
+      parser.parseEval(
+        "list($foo, $bar, $baz, list(, $nestedFoo,,, $nestedBar,)) = $array;"
+      )
+    ).toMatchSnapshot();
+  });
+
+  it("fix #174", function() {
+    expect(parser.parseEval("list($var,,,,,) = $array;")).toMatchSnapshot();
```

Comment:
in that case it's stripping only the last one, so each time prettier will pass on the file it will remove one last null until it stips every null - sounds weird

Reply:
@ichiriac hm, maybe you are right, need tests